Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/ahenry/.conda-envs/whoc/lib/python3.11/site-packages/mpi4py/futures/__main__.py", line 74, in <module>
    main()
  File "/home/ahenry/.conda-envs/whoc/lib/python3.11/site-packages/mpi4py/futures/__main__.py", line 62, in main
    run_command_line()
  File "/home/ahenry/.conda-envs/whoc/lib/python3.11/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "<frozen runpy>", line 291, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "run_case_studies.py", line 22, in <module>
    from whoc.interfaces.controlled_floris_interface import ControlledFlorisModel
ModuleNotFoundError: No module named 'whoc.interfaces'
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
